### DISCUSSION TITLE
Using Quick lists does looses cursor focus

### DISCUSSION THREAD

**Discussion by meny** [mucommander/mucommander#929] [Q&A]
Not a major issue, but very annoying:

When using Quick lists, such as Option-1, Option-2, Option-3 to jump quickly among paths, the cursor gets stuck somewhere else so that in order to regain my navigation with keyboard, I need to first press Tab to get into the file listing pane and continue.

Thanks!

**Comment by meny** [comment]
+1 

and my workaround is to actually quit every time :))

**Reply by ahadas** [reply]
@meny this reminds me of another issue that was reported about the focus that is being lost, which was closed with [a comment saying that it doesn't reproduce after upgrading macOS](https://github.com/mucommander/mucommander/issues/1358) - which operating system do you run muCommander on? can you record a video that demonstrates the issue?
